iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ValidationLevel.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public enum ValidationLevel {
+    STATE_ESTIMATION,
```

Comment:
As explained to @geofjamg the distinction between levels `STATE_ESTIMATION` and `LOADFLOW` is not understandable and relevant for me. A network after state estimation is electronically coherent. I think that the level where we don't have all the steady state hypothesis is more a `SCADA` level. After SCADA, we collect from different sources the hypothesis needed to run a state estimation.